src/Gui/ViewProviderCoordinateSystem.cpp

```diff
@@ -207,21 +220,24 @@ void ViewProviderCoordinateSystem::resetTemporarySize()
     });
 }
 
-void ViewProviderCoordinateSystem::updateData(const App::Property* prop) {
-    auto* jcs = dynamic_cast<App::LocalCoordinateSystem*>(getObject());
-    if(jcs) {
-        if (prop == &jcs->Placement) {
-            // Update position
-        }
-    }
+void ViewProviderCoordinateSystem::updateData(const App::Property* prop)
+{
+    //if (auto* lcs = getObject<App::LocalCoordinateSystem>()) {
+    //    if (prop == &lcs->Placement) {
+    //        // Update position
+    //    }
+    //}
```

Comment:
I cannot remember why there's this empty block. I think it was perhaps already there before I refactored the original 'origin' object.
So yes you can probably remove it.